challenge_3/python/ekstrah/src/a.py

```diff
@@ -0,0 +1,18 @@
+def findingMajoList(aList): #{
+    bList = set(aList)
+    bList = list(bList)
+    for i in range(0, len(bList)): #{
+        if aList.count(bList[i]) > len(bList): #{
```

Comment:
Because you are already iterating over the list and the .count() function loops over it again your overall complexity becomes O(n^2) take a look at the [Counter](https://docs.python.org/3/library/collections.html#collections.Counter) class and see if you can find a way to reduce it to O(n)